### DISCUSSION TITLE
Support of belongsToMany relationships ?

### DISCUSSION THREAD

**Discussion by guillaumeboussion** [LastDragon-ru/php-packages#14] [Ideas]
HI! First, thank's a lot for this package, needed by a lot of people using lighthouse. 
For our use, we'd like to be able to sort by a pivot table's column. Assuming this feature is only supporting one to one relation, how long would it take to implement manyToMany relationships ? I'm asking to think about implementing that by myself, but at the moment, I did not take a look into the code. I would appreciate a lot having details about the complexity of this feature !

**Comment by LastDragon-ru** [comment]
Thanks. Support of `BelongsToMany` and other `*ToMany` should be easy (it missed because the first version used joins so it was too difficult to join only one row) and actually I've planned to look on it :) (as a possible workaround for #11) But probably it will require addition relation for pivots properties because I've no idea how to specify pivot table in the grapqh query.

**Comment by LastDragon-ru** [comment]
@guillaumeboussion support of [*-to-many relations](https://github.com/LastDragon-ru/lara-asp/blob/master/packages/graphql/README.md#eloquentdatabase) available in the `master` branch 🎉
